lib/ansible/modules/uri.py

```diff
@@ -770,8 +769,7 @@ def main():
                 js = json.loads(u_content)
                 uresp['json'] = js
             except Exception:
-                if PY2:
-                    sys.exc_clear()  # Avoid false positive traceback in fail_json() on Python 2
+                ...
```

Comment:
Why not just use `with contextlib.suppress(Exception):` instead of a try-except-pass? It's a more obvious way to communicate what it does.